docs/conflib/get_simp_version.py

```diff
@@ -45,9 +45,17 @@ def get_simp_version(rootdir=ROOTDIR, simp_github_raw_base=SIMP_GITHUB_RAW_BASE,
         'simp_branch': None
     }
 
+    if os.environ.get('SIMP_VERSION', False):
```

Comment:
* I get why you did this.  However, I am concerned about the already too many confusing options to get to this information (SIMP_CORE_PATH and SIMP_BRANCH).  It doesn't help that there is code spread over the Rakefile and Python files, the nuance of whether on RTD or not, the nuance of when a `build/rpm_metadata/release` file is generated (processed in the final `else` below),  and a silly option (SIMP_BRANCH)  that doesn't build the version of docs downloaded, only uses the `simp.spec` file from that `simp-core` branch to extract the SIMP version. Can we simplify instead of adding complexity? 
* If you intend to keep this new env variable, please update the unit test for the code changes.